packages/plugins/experimental/plugin-script/src/components/ScriptToolbar/ScriptToolbar.tsx

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import React, { useCallback } from 'react';
+
+import { type ScriptType } from '@dxos/functions';
+import { fullyQualifiedId } from '@dxos/react-client/echo';
+import { ElevationProvider, type ThemedClassName } from '@dxos/react-ui';
+import { stackItemContentToolbarClassNames } from '@dxos/react-ui-editor';
+import { createGapSeparator, MenuProvider, ToolbarMenu, useMenuActions } from '@dxos/react-ui-menu';
+
+import { createDeploy } from './deploy';
+import { createFormat } from './format';
+import { createTemplateSelect } from './templateSelect';
+import { useToolbarAction } from './useToolbarAction';
+import { type ScriptToolbarState } from './useToolbarState';
+import { createView } from './view';
+
+export type ScriptToolbarProps = ThemedClassName<{
+  role?: string;
+  script: ScriptType;
+  state: ScriptToolbarState;
+}>;
+
+const createToolbar = (state: ScriptToolbarState) => {
+  const templateSelect = createTemplateSelect();
+  const format = createFormat();
+  const gap = createGapSeparator('gap');
+  const deploy = createDeploy(state);
+  const view = createView(state);
+  return {
+    nodes: [...templateSelect.nodes, ...format.nodes, gap, ...deploy.nodes, ...view.nodes],
+    edges: [
+      ...templateSelect.edges,
+      ...format.edges,
+      { source: 'root', target: gap.id },
```

Comment:
couldn't we have a shortcut for this? no need for the user to create a new instance?